### DISCUSSION TITLE
arm server

### DISCUSSION THREAD

**Discussion by Mohammadreza5196** [penumbra-zone/penumbra#4161] [Q&A]
hello dear
how do i run pcli on arm server. i have an error when run it on my server.

bash: /usr/local/bin/pcli: cannot execute binary file: Exec format error

**Comment by conorsch** [comment]
@Mohammadreza5196 We only build binaries for specific platforms as part of the release objects:

* x86 linux
* x86 mac
* m1 mac

See for example here: https://github.com/penumbra-zone/penumbra/releases/tag/v0.71.0 If you need support for another architecture, you'll have to compile from source yourself. See our guide https://guide.penumbra.zone/main/dev/build.html
